webpack/components/PreupgradeReports/components/__snapshots__/EntriesFilter.test.js.snap

```diff
@@ -83,10 +83,13 @@ exports[`EntriresFilter should render for host filter 1`] = `
               Inhibitor
             </MenuItem>
           </DropdownButton>
-          <FormControl
-            bsClass="form-control"
-            className="entries-filter-reset-height"
-            componentClass="input"
+          <DebounceInput
+            className="form-control entries-filter-reset-height"
+            debounceTimeout={300}
+            element="input"
+            forceNotifyByEnter={true}
```

Comment:
From react-input-debounce docs

> Notification of current value will be sent immediately by hitting Enter key.

When I type something in and hit enter, it triggers a full page reload, taking me back to the job invocation overview. Is this on our side or in the library?

Reply:
Good catch, that one is actually on me. It should be fixed now.